src/components/Icon/Icon.tsx

```diff
@@ -21,6 +22,7 @@ type IconSize = 3 | 4 | 5 | 6 | 7 | 8 | 9;
  * @param {boolean} [ariaHidden]          Determines whether or not the element is hidden from a screen reader.
  * @param {string}  [role]                Describes the role of an element in programs that can make use of it
  * @param {string}  [ariaLabel]           Error message text displayed when inputError === true.
+ * @param {boolean} [renderWhiteIcon]      Renders a white icon instead of the default black icon.
```

Comment:
Do we need to add a param for color here? I understand color is coming from IntrinsicElements, but because we're including it in the logic of line 44 should that be documented?